lib/sentry/config.ex

```diff
@@ -0,0 +1,89 @@
+defmodule Sentry.Config do
```

Comment:
@mitchellhenke no, its working fine. I meant, please don't change the current, working behaviour :)